src/apps/reservation-list/list/reservation-pause-toggler.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState, FC, useEffect } from "react";
+import ReservationsIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/Reservations.svg";
+import { getModalIds } from "../../../core/utils/helpers/general";
+import { useModalButtonHandler } from "../../../core/utils/modal";
+import { useText } from "../../../core/utils/text";
+import PauseReservation from "../modal/pause-reservation/pause-reservation";
+import { PatronV5 } from "../../../core/fbs/model";
+
+interface ReservationPauseTogglerProps {
+  user: PatronV5;
+}
+
+const ReservationPauseToggler: FC<ReservationPauseTogglerProps> = ({
+  user
+}) => {
+  const t = useText();
+  const { open } = useModalButtonHandler();
+  const { pauseReservation } = getModalIds();
+  const [onHoldDates, setOnHoldDates] = useState<string>();
+
+  const openPauseReservationModal = () => {
+    open(pauseReservation as string);
+  };
+
+  useEffect(() => {
+    if (user && user.onHold) {
+      setOnHoldDates(`${user.onHold.from} - ${user.onHold.to}`);
```

Comment:
Don't you need `onHoldDates` in the dependency array if `useEffect`?
Could be that you have an infinite loop now because of it. Also I would set the initial state of `onHoldDates ` to `null` and add a check for it in:
```
if (!onHoldDates  && user && user.onHold) {

Reply:
I dont think I understand, why do I need onHoldDates in the dependency array?